doc/rados/configuration/mon-config-ref.rst

```diff
@@ -591,7 +591,7 @@ Trimming requires that the placement groups are ``active + clean``.
 
 ``paxos trim disabled max versions``
 
-:Description: The maximimum number of version allowed to pass without trimming.
+:Description: The maximum number of version allowed to pass without trimming.
```

Comment:
this option was removed in ccceeee57b4b31709a79b146c5075ef2a909568d.

and "paxos trim tolerance" was removed in 6b8e74f0646a7e0d31db24eb29f3663fafed4ecc also.

could you remove both of them in the doc?